src/main/resources/META-INF/plugin.xml

```diff
@@ -94,13 +94,13 @@
   <change-notes><![CDATA[
     <h2>New Features</h2>
       <ul>
-        <li>Replaced Activity Alert Dialog with a Notification Balloon which is non-blocking.</li>
-      </ul>
-    <h2>Bug Fixes</h2>
-      <ul>
-        <li>Fixed repeated activity alert being shown to some users.</li>
-        <li>Fixed populateBuffer issue for unopened synced project with new branch.</li>
+        <li>Updated file sync message from "Syncing Files" to "CodeSync: Syncing Files..."</li>
```

Comment:
Do we have to remove previous content?

Reply:
I assumed that this modified changelog was going to be associated with a new release. Considering that I removed the previous content

Reply:
What's the best practice? Should we keep previous notes somehow and add new for every release?

Reply:
Ideally, I think we should keep previous notes somewhere for record keeping purposes. Maybe add them in a table? I was also considering maybe using a dict to store versions, but i don't think that would be feasible considering there can be x number of new versions

patch_notes = {
    '1.8': {
        'New Features': [
            'Replaced Activity Alert Dialog with a Notification Balloon which is non-blocking.'
        ],
        'Bug Fixes': [
            'Fixed repeated activity alert being shown to some users.',
            'Fixed populateBuffer issue for unopened synced project with new branch.'
        ]
    },
    '1.7': {
        'New Features': [
            'Added XYZ feature.',
            'Improved ABC functionality.'
        ],
        'Bug Fixes': [
            'Fixed issue with XYZ.',
            'Addressed ABC bug.'
        ]
    }
}

Reply:
@Basit-qc if this will be displayed on the marketplace then I think it would be better to only include the changes that are part of the current release and remove previous ones. I have seem similar behavior for mobile apps on the app stores.

Reply:
Yes it appears on the marketplace, so I think it is fine to remove since we already have CHANGELOG file for all changes, fyi @saadberry